### DISCUSSION TITLE
Localhost not  working

### DISCUSSION THREAD

**Discussion by emrik00** [TGRHavoc/live_map-interface#54] [Q&A] [answered]
Everything works for me.  I have port-forwarded my server, and the 30121 port. It work's completely fine, just not for other people. Is there any solution  to this? Also I can't really code well, could you maybe send the client.lua just that no-one else will show up else than people in policecars? I know you haven't really made this for RP servers, but I would really like to could do it. I can't really code:P

**Comment by TGRHavoc** [comment, marked as answer]
@0pax you need to use the server's public IP in the config.json file for it to work for others.
